api/src/pcapi/notifications/push/backends/logger.py

```diff
@@ -36,3 +36,6 @@ def delete_user_attributes(self, user_id: int) -> None:
             "A request to delete user attributes would be sent for user with id=%d",
             user_id,
         )
+
+    def send_transactional_notification_delayed(self, notification_data: TransactionalNotificationData) -> None:
+        self.send_transactional_notification(notification_data)
```

Comment:
(style / question de goût) variante plus concise (et plus clair, mais c'est peu-être subjectif)

```suggestion
    send_transactional_notification_delayed = send_transactional_notification
```